### ISSUE TITLE
[4.0] HTML on Locked-Article

### ISSUE THREAD

**Issue by ghost** [joomla/joomla-cms#21373]
### Steps to reproduce the issue
Hover the Locked-Icon get Information using `<strong>, <br>`:

![screen shot 2018-08-03 at 06 08 23](https://user-images.githubusercontent.com/8235763/43623925-4b31b680-96e4-11e8-9065-5a653dad1b40.jpg)
### System information
- Nightly Build 4.0.0-alpha5-dev
- Multilingual Sample Data (French, German DE, Persian)
- Template: Cassiopeia
- macOS Sierra, 10.13.6
- Firefox 61 (64-bit)
#### CloudAccess.net
- PHP 7.1.15
- MySQLi 5.7.18-cll-lve

**Comment by SharkyKZ** [comment]
CC @C-Lodder as author of #17050. Will 4.0 have its own version of tooltips or should Bootstrap tooltips be converted to Bootstrap popovers?

**Comment by C-Lodder** [comment]
@SharkyKZ I dont know anymore. Not sure if they're being converted to custom elements or what. Best ask @ciar4n and @dgrammatiko

**Comment by wilsonge** [comment]
Use popovers if it needs html

**Comment by alikon** [comment]
is this still an issue ?

**Comment by brianteeman** [comment]
yes

**Comment by brianteeman** [comment]
If #24441 is accepted then the technique used there could be used here as well

**Comment by Quy** [comment]
PR #23944 but still debatable whether it is the right fix

**Comment by wilsonge** [comment]
Use the tooltips css from https://github.com/joomla/joomla-cms/pull/24892

**Comment by brianteeman** [comment]
I will do that shortly

**Comment by brianteeman** [comment]
or maybe not as it relied on getting focus

**Comment by ciar4n** [comment]
Your tooltip CSS should also work for hover. Just need to include :hover....

```
 /* reveal associated tooltip on focus */
:focus + [role="tooltip"],
:hover + [role="tooltip"] {
  display: block;
  position: absolute;
  bottom: 100%;
}
```

**Comment by brianteeman** [comment]
yes I have it working on hover but i havent been able to get focus to work - it needs focus for keyboards

**Comment by ciar4n** [comment]
At a glance, if your CSS is the same as the above then the focus should work IF the hover works.

**Comment by brianteeman** [comment]
I agree it should but it appears there is a difference between what can be focussed and hovered. I will get there in the end.

**Comment by brianteeman** [comment]
Got it

**Comment by wilsonge** [comment]
@brianteeman What ended up happening here?

**Comment by brianteeman** [comment]
see #24899

**Comment by ghost** [comment]
Closed as having Pull Request #24899
